### PR TITLE
Rework: Formatting, some wording added link to video-tutorial

### PR DESCRIPTION
**Pull request by GeraldIr**

### CODE REVIEW

File: workspaces/index.md [hunk 1]
```diff
@@ -1,6 +1,239 @@
-# Workspaces Overview
+# Workspaces
 
-openEO Workspaces are a new concept introduced to the openEO specification, 
-they allow the users to have more control over where their data is saved to, loaded from or stored.
+## Workspaces Overview
 
-All the code-snippets and more can be found on [GitHub](https://github.com/eodcgmbh/eodc-examples/blob/main/demos/workspaces/).
\ No newline at end of file
+Workspaces are an abstraction of object-storage for openEO. They allow you to have more control over their data, from where it is saved, to where it is loaded from.
```
Comment (SerRichard) [inline workspaces/index.md:5]: Should probably be either "OpenEO Platform" or "the OpenEO Api"

Comment (SerRichard) [inline workspaces/index.md:5]:
```suggestion
Workspaces are an abstraction of object-storage for openEO. They allow you to have more control over your data, from where it is saved, to where it is loaded from.
```

File: workspaces/index.md [hunk 2]
```diff
@@ -1,6 +1,239 @@
...
+
+```python
+import json
+import requests
+
+response = requests.get(
+    url="https://openeo.example.eu/openeo/1.2.0/workspace_providers",
+)
+
+print(json.dumps(json.loads(response.content.decode()), indent=4, sort_keys=True))
```
Comment (SerRichard) [inline workspaces/index.md:33]: could response.json() be used here?

**Review by SerRichard — APPROVED** [review]
Read through, lgtm, a couple of minor comments for possible changes.

File: workspaces/index.md (hunk 2 above)
Comment (GeraldIr) [reply workspaces/index.md:31]: No, sadly.

**Pull request merged**